sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/PercentileSuite.scala

```diff
@@ -119,35 +174,73 @@ class PercentileSuite extends SparkFunSuite {
   test("call from sql query") {
```

Comment:
Can you remove this entire test. We don't have to check the `sql` method. This feels like over testing.